app/design/frontend/rwd/default/template/catalog/product/view/sharing.phtml

```diff
@@ -27,7 +27,7 @@
     <?php $_productImageUrl = urlencode(trim($this->helper('catalog/image')->init($_product, 'image')))?>
     <?php $_productUrl = urlencode(trim($_product->getProductUrl()))?>
     <li>
-        <?php $_u = 'p[url]=' . $_productUrl . '&p[images][0]=' . $_productImageUrl . '&p[title]=' . $_productName . '&p[summary]=' . urlencode(trim($_helper->productAttribute($_product, nl2br($_product->getShortDescription()), 'short_description'))); ?>
+        <?php $_u = 'p[url]=' . $_productUrl . '&p[images][0]=' . $_productImageUrl . '&p[title]=' . $_productName . '&p[summary]=' . urlencode(trim($_helper->productAttribute($_product, nl2br($_product->getShortDescription() ?? ''), 'short_description'))); ?>
```

Comment:
```suggestion
        <?php $_u = 'p[url]=' . $_productUrl . '&p[images][0]=' . $_productImageUrl . '&p[title]=' . $_productName . '&p[summary]=' . urlencode(trim($_helper->productAttribute($_product, $_product->getShortDescription(), 'short_description'))); ?>
```

Because `nl2br()` is duplicated and should be fixed here:
https://github.com/OpenMage/magento-lts/blob/7a3e76522b5b519786f9594a16c1783a9de50e2c/app/code/core/Mage/Catalog/Helper/Output.php#L126

Reply:
@kiatng this line is not called.

See line 120: `!$attribute->getIsHtmlAllowedOnFront()`